_tasks/codegen.ts

```diff
@@ -0,0 +1,21 @@
+import { LocalCapiCodegenServer } from "../codegen/server/local.ts"
+import * as fs from "../deps/std/fs.ts"
+
+await fs.emptyDir("target/codegen/generated")
+const port = 5646
+const server = new LocalCapiCodegenServer()
```

Comment:
Should `LocalCodegenServer` (and its ancestors) accept the signal via props (as does `serve` from `http/server.ts`)?

Reply:
I originally wrote it that way, but found it annoying to create a new abort controller every time.